webpack.config.js

```diff
@@ -24,3 +22,30 @@ module.exports = {
     ],
   },
 };
+
+export default [
+  {
+    ...defaultConfig,
+    output: {
+      path: path.resolve(__dirname, './dist'),
+      filename: 'ods-charts.js',
+      library: {
+        type: 'umd',
+        name: 'ODSCharts',
+      },
+    },
+  },
+  {
+    ...defaultConfig,
+    output: {
+      path: path.resolve(__dirname, './dist'),
+      filename: 'ods-charts-module.js',
```

Comment:
Usually, the naming convention used by libraries is the following:
* `ods-charts.js`
* `ods-charts.esm.js`
* `ods-charts.umd.js`

And for the minified versions:
* `ods-charts.min.js`
* `ods-charts.esm.min.js`
* `ods-charts.umd.min.js`